xrviz/dashboard.py

```diff
@@ -202,3 +230,14 @@ def check_is_plottable(self, var):
         self.plot_button.disabled = False  # important to enable button once disabled
         data = self.data[var[0]]
         self.plot_button.disabled = len(data.dims) <= 1
+
+    def cartopy_installed(self):
+        try:
+            from cartopy import crs as ccrs
+            import geoviews as gv
+            import geoviews.feature as gf
+            global ccrs, gv, gf
+            self.has_cartopy = True
+        except:
+            print("Install Cartopy to view Projection Panel.")
```

Comment:
This would be a warning or a logging command, never a print. Only use print for short-term debugging.

Reply:
Corrected. Now using `warnings.warn`.

Reply:
We *may* get annoyed feedback on this, because users without cartopy will see this every time. Might need to change later.